config.json

```diff
@@ -193,6 +193,12 @@
 			"pattern": "^(did:orb:.+)$",
 			"url": "http://orb-did-driver:8121/resolveDID?did=$1",
 			"testIdentifiers": [ "did:orb:hl:uEiDI6sZJnp44sC_BG2XekWA4TN2m0Wtwv8AI71UER3vCHw:uoQ-BeEJpcGZzOi8vYmFma3JlaWdpNWxkZXRodTZoY3ljN3FpM214cGpjeWJ5anRvMm51bGxvYzc0YWNocGt1Y2VvNjZjZDQ:EiClaNSwhKSP6tQe36bYc5w41oRpCi1xv89MkUnNBm8agw" ]
+		},
+    {
+			"pattern": "^(did:dock:[a-zA-Z0-9]+)$",
+			"image": "docknetwork/dock-did-driver",
```

Comment:
The internals changed a bit since your driver was removed. The `image` property was replaced by `url` which is used to invoke the driver. See the [docs](https://github.com/decentralized-identity/universal-resolver/blob/main/docs/driver-development.md#driver-interface) and other driver examples.

Reply:
This may have been a merge conflictafter fetching from upstream, will update, also see tabulation is off